CONTRIBUTING.md

```diff
@@ -1,15 +1,27 @@
-# How to contribute
+# Contribution Guide #
 
-This document outlines some of the conventions on development workflow, commit message formatting, contact points and other
+TiDB is a community driven open source project. All contributor are very welcome. The process of contributing to the TiDB project
+may be different than many other projects you get involved in. This document outlines some conventions about development workflow, commit message formatting, contact points and other
 resources to make it easier to get your contribution accepted.
 
-## Getting started
+## Becoming a contributor ##
 
-- Fork the repository on GitHub.
-- Read the README.md for build instructions.
-- Play with the project, submit bugs, submit patches!
+Before you can contribute the TiDB project, you need setup a few prerequisites. 
 
-## Contribution flow
+### Contributor License Agrement ###
+Before sending your first change to TiDB, you have to sign the CLA. PingCap only accepts the [individual contributor license agreement](https://cla-assistant.io/pingcap). 
+This means if your organzation is the copyright holder, you can not contribute to this project. 
```

Comment:
Before pulling your first request to TiDB, you have to sign the CLA. It is noted that PingCAP only accepts the [individual contributor license agreement](https://cla-assistant.io/pingcap), which means if your organization is the copyright holder, you can not contribute to this project.